geometry_msgs/Point32 P
float32[9] R
geometry_msgs/Point32 T
---
geometry_msgs/Point32 P_prime

